<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectToolMenu</class>
 <widget class="QWidget" name="selectToolMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>67</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Brush size: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sizeSpinBox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Free fill sensitivity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fillSensSpinBox">
       <property name="toolTip">
        <string>How sensitive is free fill. A value of 0% would mean only a singular color is filled, while 99% woul mean almost no restrictions. 5% is recomended.</string>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Free subtraction sensitivity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="subSensSpinBox">
       <property name="toolTip">
        <string>How sensitive is free subtraction. A value of 0% would mean only a singular color is subtracted, while 99% woul mean almost no restrictions. 3% is recomended.</string>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>-3</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Free fill distance: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="distDoubleSpinBox">
       <property name="toolTip">
        <string>Distance times brush size that the algorithm will perform free fill. If set to 1.0, no free fill will occur.</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
